bridge/tracker/tracker.go

```diff
@@ -0,0 +1,390 @@
+package tracker
+
+import (
+	"context"
+	"encoding/json"
+	"math/big"
+	"os"
+
+	"github.com/0xPolygon/polygon-edge/types"
+
+	"github.com/hashicorp/go-hclog"
+	"github.com/syndtr/goleveldb/leveldb"
+	"github.com/umbracle/go-web3"
+)
+
+const (
+	lastQueriedBlockNumber = "last-block-num" //	db key
+)
+
+//	Tracker represents an event listener that notifies
+//	for each event emitted on the rootchain (ethereum).
```

Comment:
Why is `Ethereum` here? This can theoretically be used for any blockchain network that supports the formats we're expecting

Reply:
[Removed](https://github.com/0xPolygon/polygon-edge/pull/386/commits/c3a1d5a0b102dd34858726ae9e6fcf38cfe62204). Although, the discussion for our expected header format is open.